src/main/java/net/ossindex/version/parser/Version.g4

```diff
@@ -93,7 +93,8 @@ union_range
 logical_range
 	: '(' logical_range ')'
 	| '(' simple_range ')'
-	
+	| '(' union_range ')'
```

Comment:
@ken-duck  Couldn't figure out why 'union_range' is added

Reply:
Union range is added because it is the one that provides support for "range1 | range2". We need it added here because this allows us to nest bracketed union ranges within logical ranges, which are the "range1 & range2".

Reply:
Basically it adds support for oddly bracketed ranges.